Gemfile.lock

```diff
@@ -0,0 +1,57 @@
+PATH
```

Comment:
I got this after I ran `bundle install`.

A `Gemfile.lock` ensures everyone doing development on this application uses the same exact set of dependencies (you, your colleagues, and GitHub Actions).

Note that when another user uses this gem, they will create their **own** `Gemfile.lock` based on the **loose** version constraints in the `gemspec`.

Bundler has some amazing documentation on gem versioning, `Gemfile.lock` files, and more.

https://bundler.io/guides/faq.html